src/controllers/funds.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-console */
```

Comment:
please refrain from disabling this rule. A best way would be to define a logger function and use it in dev environment only.